src/containers/SideBar/components/Settings/DebugSettings.tsx

```diff
@@ -1,14 +1,13 @@
 import { Typography } from '@app/components/elements/Typography.tsx';
 import { useTranslation } from 'react-i18next';
 import calculateTimeSince from '@app/utils/calculateTimeSince.ts';
-import { useAppStatusStore } from '@app/store/useAppStatusStore.ts';
 import { Stack } from '@app/components/elements/Stack.tsx';
-import { useShallow } from 'zustand/react/shallow';
+import { useMiningStore } from '@app/store/useMiningStore';
 
 export default function DebugSettings() {
     const { t } = useTranslation(['common', 'settings'], { useSuspense: false });
 
-    const blockTime = useAppStatusStore(useShallow((state) => state.base_node?.block_time));
+    const blockTime = useMiningStore((state) => state.base_node?.block_time);
```

Comment:
not sure if there's it's an issue with the store/setting the value, but the settings counter doesn't updaye

https://github.com/user-attachments/assets/47284f3f-8525-4575-8bfb-3f88c345ce65

Reply:
Fixed

Reply:
Hey, it doesn't work on current main.